cost/google/old_snapshots/google_delete_old_snapshots.pt

```diff
@@ -170,18 +179,30 @@ end
 policy "policy_old_snapshots" do
   validate_each $ds_filtered_snapshots do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows containing Old Snapshots"
-    detail_template <<-EOS
-# Google Old Snapshots
-| Project ID | Name | Size | Labels | Age In Days| Status |
-| ---------- | ---- | ---- | ------ | ------------- | ------ |
-{{ range data -}}
-| {{.projectId}} | {{.name}} | {{.diskSizeGb}} | {{ range $i, $e := .labels }}{{$i}}:{{ $e }}, {{ end }} | {{.days_old}} | {{.status}} |
-{{ end -}}
-###### Policy Applied in Account: {{ rs_project_name }} (Account ID: {{ rs_project_id }}) within Org: {{ rs_org_name }} (Org ID: {{ rs_org_id }})
-EOS
     check lt(dec(now,to_d(val(item, "creationTimestamp"))), prod($param_snapshot_age, 24*3600))
     escalate $esc_email
     escalate $esc_approve_delete_snapshots
```

Comment:
The data for region/zone should be added,

Reply:
@cdwilhelm, We are not getting "Zone" in the response [document : https://cloud.google.com/compute/docs/reference/rest/v1/snapshots/get#response-body]. 

**When I applied the policy I'm getting the following exception:**
Message:  to_d: could not convert nil to datetime
Location:
  function "lt"
    argument #1
      function "dec"
        argument #2
          function "to_d"

Reply:
have you examined the policy to see what might be causing the error?  What have you done to understand the issue?

Reply:
@cdwilhelm, The policy is failing because in one of the responses the snapshot createTime is null so we are getting the exception. I have added the null check. Thanks.